bliss/simulator/prior.py

```diff
@@ -86,18 +99,73 @@ def __init__(
         self.galaxy_a_scale = galaxy_a_scale
         self.galaxy_a_bd_ratio = galaxy_a_bd_ratio
 
-    def sample_prior(self) -> TileCatalog:
+        # load all star, galaxy ratios required for sampling
+        if self.n_bands == 1:
+            return
+        for field_params in sdss_fields["field_list"]:
+            run = field_params["run"]
+            camcol = field_params["camcol"]
+            fields = field_params["fields"]
+            for field in fields:
+                sdss_path = Path(self.sdss)
+                camcol_dir = sdss_path / str(run) / str(camcol) / str(field)
+                po_path = camcol_dir / f"photoObj-{run:06d}-{camcol:d}-{field:04d}.fits"
+                po_fits = fits.getdata(po_path)
```

Comment:
A lot of the codecov warnings can probably be fixed by adding a test that uses SDSS backgrounds drawn from multiple fields. I can work on writing one this afternoon/evening.

Reply:
Yep, it'd be good add to a test to keep our coverage from decreasing

Reply:
Added tests for initializing simulator with multiple fields for background/psf, and with multiple bands